packages/plugin-js-packages/src/lib/runner/outdated/unify-type.ts

```diff
@@ -0,0 +1,40 @@
+import { objectToEntries } from '@code-pushup/utils';
+import {
+  NormalizedVersionOverview,
+  NpmOutdatedResultJson,
+  OutdatedResult,
+  Yarnv1OutdatedResultJson,
+} from './types';
+
+export function npmToOutdatedResult(output: string): OutdatedResult {
+  const npmOutdated = JSON.parse(output) as NpmOutdatedResultJson;
+  // current might be missing in some cases
+  // https://stackoverflow.com/questions/42267101/npm-outdated-command-shows-missing-in-current-version
+  return objectToEntries(npmOutdated)
+    .filter(
+      (entry): entry is [string, NormalizedVersionOverview] =>
+        entry[1].current != null,
+    )
+    .map(([name, overview]) => ({
+      name,
+      current: overview.current,
+      latest: overview.latest,
+      type: overview.type,
+      project: overview.dependent,
```

Comment:
Is the `dependent` field useful to show? Looks like it produces `cli` for our entire repo.

Reply:
Thought it might use actual project name but seeing that only the root `git` folder is used, probably not.